### PR TITLE
Add card web component to vet center featured-content

### PR DESCRIPTION
**Pull request by jtmst**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

-Modifies the featured content component for vet centers to use the va-card web component.
- Some style classes modified for new card styling
- 

## Related issue(s)

-  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17618

## Testing done

- Test suite ran locally
- Local visual testing

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

image

image

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

urls used for testing:
- `/boston-health-care/medical-records-office/`
- `/pittsburgh-vet-center/`

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by laflannery** [comment]
@jtmst, @thejordanwood and I met so this is our combined feedback:

We want to remove some of the custom spacing and sizing that was present in the existing implemention:
- Can you remove all the `vads-u-padding` classes on each card? This will putthe standard 16px padding back
Screenshot 2024-05-06 at 3 08 48 PM
- Can you remove the `force-small-header` class on the H3s?
- Can you also remove the bottom margin but add `vads-u-margin-top--0` to the H3s? This would now look like this:
Screenshot 2024-05-06 at 3 11 41 PM

The new cards should look like this with those changes
Screenshot 2024-05-06 at 2 24 53 PM

**Comment by jilladams** [comment]
Laura threw a Change Management flag: https://dsva.slack.com/archives/C0FQSS30V/p1715023108494009.  Marking PR DNM til we discuss.

File: src/site/includes/vet_centers/featured_content.liquid [hunk 1]
```diff
@@ -1,23 +1,23 @@
-<div class="feature featured-content-list-item vads-u-flex--fill vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--0 medium-screen:vads-u-margin-bottom--2">
```
Comment (randimays) [inline src/site/includes/vet_centers/featured_content.liquid:1]: Was there corresponding CSS that needs to be removed for `feature`, `featured-content-list-item` and `force-small-header`?

**Review by randimays — COMMENTED** [review]
Left one comment. For your PRs, could you include the URLs you used to test, please?

**Review by thejordanwood — APPROVED** [review]

**Comment by thejordanwood** [comment]
Looks good to me!

**Comment by jilladams** [comment]
CM comms have gone out: https://dsva.slack.com/archives/C0FQSS30V/p1715098254352709. Unblocked!

File: src/site/includes/vet_centers/featured_content.liquid (hunk 1 above)
Comment (jtmst) [reply src/site/includes/vet_centers/featured_content.liquid:1]: Looks like those classes are used elsewhere at the moment, so I dont want to remove the CSS.  And good call, will add URLs to the PR body

Comment (randimays) [reply src/site/includes/vet_centers/featured_content.liquid:1]: Great, thanks for checking!

**Review by randimays — APPROVED** [review]
LGTM!

**Review by eselkin — APPROVED** [review]
LGTM

**Pull request merged**